content/400-pulse/600-faq.mdx

```diff
@@ -60,10 +60,39 @@ No, Pulse is server-side and subscriptions cannot be initiated directly within c
 
 To propagate events to the frontend, you can use a WebSocket library like [socket.io](https://socket.io/).
 
-## How to configure `tsconfig.json` with the Pulse extension?
+## How do I import the Pulse Client extension when using `"moduleResolution":"bundler"` in my `tsconfig.json`?
 
-If you're using [`"moduleResolution": "bundler"`](https://www.typescriptlang.org/docs/handbook/modules/reference.html#bundler) in your `tsconfig.json` file, you need to
-explicitly set [`customConditions`](https://www.typescriptlang.org/tsconfig/#customConditions) to either `node` or `workerd` depending on your target runtime. This will instruct TypeScript to match the correct type definitions of the Prisma Pulse extension, as well as any other packages that expose multiple entrypoints:
+If you're using [`"moduleResolution": "bundler"`](https://www.typescriptlang.org/docs/handbook/modules/reference.html#bundler) in your `tsconfig.json` file, and your Pulse client extension version is `1.2.0` or greater, you need to import the Pulse Client extension in a way that matches your target runtime environment:
+
+<TabbedContent code>
+
+<TabItem value="Node.js">
+
+```ts
+import { withPulse } from '@prisma/extension-pulse/node';
+```
+
+</TabItem>
+
+<TabItem value="Cloudflare Workers">
+
+```ts
+import { withPulse } from '@prisma/extension-pulse/workerd';
+```
+
+</TabItem>
+
+</TabbedContent>
+
+This is needed because Prisma Pulse extension offers separate implementations tailored for various runtimes, such as Node.js and Cloudflare Workers. This will instruct TypeScript to match the correct type definitions of the Prisma Pulse extension, as well as any other packages that expose multiple entrypoints.
+
+This runtime-specific import approach also resolves the following error:
+
+```
+Cannot find module '@prisma/extension-pulse' or its corresponding type declarations.
+```
+
+Alternatively, you can ensure the types are working for the target-runtime by configuring your `tsconfig.json` file and explicitly setting [`customConditions`](https://www.typescriptlang.org/tsconfig/#customConditions) to either `node` or `workerd` depending on your target runtime:
```

Comment:
Is this similar to adding this line to your tsconfig? Or is this a completely different thing. Just want to make sure we aren't removing this or forgetting to add it https://github.com/prisma/prisma-examples/blob/bbab062cce9d2aeb45911beb92f520f98daf5cb5/pulse/fullstack-leaderboard/tsconfig.json#L10C4-L10C35

Reply:
@mhessdev it's another way of doing the same thing. We felt that this would be more intuitive for our users.